src/sidebar/components/menu.js

```diff
@@ -126,14 +126,16 @@ export default function Menu({
         onClick={toggleMenu}
         title={title}
       >
-        {label}
-        {menuIndicator && (
-          <span
-            className={classnames('menu__toggle-arrow', isOpen && 'is-open')}
-          >
-            <SvgIcon name="expand-menu" className="menu__toggle-icon" />
-          </span>
-        )}
+        <span className="menu__toggle-wrapper">
```

Comment:
This was the magic sauce, I needed to move display: flex off `.menu__toggle` so it would behave like a flex item, not a flex container. But we need still need a flex lay out for both the label and menuIndicator stuff so wrapping it in another container was required.

Reply:
Given that this explanation is crucial for understanding the changes in this PR, it might be a good idea to put some commenting around the CSS for this wrapper `span` to elucidate future devs, or create a commit body with the same information so that it will be available in the future.

Reply:
Good idea, I'll add a comment in code, its a bit harder to fid commit logs

Reply:
I should be a bit more clear, it was part of the magic sauce! W/o this change, the button didn't fill the parent container and match the height of the publish button. It was actually not even apparent until I started scaling the browser with the CMD+ and I noticed it was off by 1 pixel in height. This gif I made shows the problem more dramatically as I increase the height from 35 or 65 
![layout](https://user-images.githubusercontent.com/3939074/73093359-32b41a00-3e93-11ea-9053-74d718b9d6f0.gif)